iroha_telemetry/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+//! Crate with iroha telemetry processing
+
+pub mod futures;
```

Comment:
Now we have telemetry in logger and here. If we have made a separate crate, then I suggest to move telemetry logic from logger also here.

Reply:
Tried to do that, but it turns out that it will be a circular dependency graph. I guess we can leave it like that?

Reply:
Ok, then